javascript/javascript1/week1/homework/startupNameGenerator.js

```diff
@@ -0,0 +1,35 @@
+console.log("===Startup Name Generator===")
+const firstWords = [
+  "Alpha",
+  "Beta",
+  "Cargo",
+  "Dango",
+  "Echo",
+  "FisherMan",
+  "Gokyo",
+  "Helicoptor",
+  "InkMan",
+  "Romeo",
+];
+const seconWords = [
+  "Floppy",
+  "Luck",
+  "Star",
+  "Mango",
+  "Friendly",
+  "Ship",
+  "Trek",
+  "Smash",
+  "Rider",
+  "Juliet",
+];
+
+function getRandomNumber() {
+  const randomNumber = Math.floor(Math.random() * 10);
+  return randomNumber;
```

Comment:
As you don't use the var for anything else but to return it on next line, you can write the entire thing as this:
```suggestion
  return Math.floor(Math.random() * 10);
```